### ISSUE TITLE
Helm service creation issue

### ISSUE THREAD

**Issue by saimadineni** [aws-samples/eks-workshop#632]
While I was trying to follow instructions from this page https://eksworkshop.com/beginner/060_helm/helm_nginx/installnginx/, My installation has failed 
helm install --name mywebserver bitnami/nginx

Error: release mywebserver failed: namespaces "default" is forbidden: User "system:serviceaccount:kube-system:default" cannot get resource "namespaces" in API group "" in the namespace "default"

And I've found the resolution for this issue in https://github.com/fnproject/fn-helm/issues/21.

Basically to run these three steps
kubectl create serviceaccount --namespace kube-system tiller
kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
kubectl patch deploy --namespace kube-system tiller-deploy -p '{"spec":{"template":{"spec":{"serviceAccount":"tiller"}}}}'

**Comment by fmedery** [comment]
Hello,
We upgraded the workshop for Helm V3. Commit:  #661  and #662
